kvui.py

```diff
@@ -627,4 +732,3 @@ def _handle_text(self, node: JSONMessagePart):
 if os.path.exists(user_file):
     logging.info("Loading user.kv into builder.")
     Builder.load_file(user_file)
-
```

Comment:
Python files should end with a newline character.

Reply:
This was an extra newline; the file still ends with a newline as per the style guide.